grid2op/Environment/Environment.py

```diff
@@ -550,6 +552,7 @@ def reset(self):
                                          names_chronics_to_backend=self.names_chronics_to_backend)
         self.current_obs = None
         self.env_modification = None
+        super().reset()
```

Comment:
Maybe that should be called first, what if Parent class resets the chronics_handler ?

Reply:
yep you are right. I'll fix that :-)